config/initializers/flipper.rb

```diff
@@ -41,12 +41,11 @@
 
 Flipper::UI.configuration.feature_creation_enabled = false
 # Make sure that each feature we reference in code is present in the UI, as long as we have a Database already
-FLIPPER_FEATURE_CONFIG['features'].each_key do |feature|
-  unless Flipper.exist?(feature)
-    Flipper.add(feature)
-    # default features to enabled for development and test only
-    Flipper.enable(feature) if Rails.env.development? || Rails.env.test?
-  end
+FLIPPER_FEATURE_CONFIG['features'].each do |feature, feature_config|
+  Flipper.add(feature) unless Flipper.exist?(feature)
+
+  # default features to enabled for test and those explicitly set for development
+  Flipper.enable(feature) if Rails.env.test? || (Rails.env.development? && feature_config['enable_in_development'])
```

Comment:
It seems we shouldn't be trying to enable a feature unless it exists. Is there a reason to exclude this from `.exists?` conditional?

Reply:
this was an oversight, good catch!